nomad/drainer/watch_jobs.go

```diff
@@ -144,13 +144,11 @@ func (w *drainingJobWatcher) watch() {
 	for {
 		w.logger.Trace("getting job allocs at index", "index", waitIndex)
 		jobAllocs, index, err := w.getJobAllocs(w.getQueryCtx(), waitIndex)
-		w.logger.Trace("retrieved allocs for draining jobs", "num_allocs", len(jobAllocs), "index", index, "error", err)
```

Comment:
👍 I've run into this one a bunch in local development and hadn't had the chance to dig into it yet.